frontend/src/app/components/filters/query-filter/query-filter.component.html

```diff
@@ -36,9 +36,18 @@
     [filter]="filter"></filter-boolean-value>
 
   </div>
-
+  
+  <div class="advanced-filters--filter-value" *ngIf="filter.name==='Parent'">
+    <filter-searchable-multiselect-value
+      (filterChanged)="onFilterUpdated($event)"
+      [shouldFocus]="shouldFocus"
+      [filterConditions]="parentFilter.filters"
+      [filterResource]="parentFilter.resource"
+      [filterSearchKey]="parentFilter.searchKey"
+      [filter]="filter"></filter-searchable-multiselect-value>
+  </div>
   <!-- Values -->
-  <ng-container *ngIf="showValuesInput && valueType">
+  <ng-container *ngIf="showValuesInput && valueType && filter.name!=='Parent'">
     <div class="advanced-filters--filter-value" [ngSwitch]="valueType">
```

Comment:
I see that the rest of filters are rendered base on the valueType of the filter. The Parent filter has the valueType (filter.currentSchema.values.type) of "[]WorkPackage". Couldn't we rely on it?